README.md

```diff
@@ -709,8 +709,10 @@ sops-nix also provides a home-manager module.
 This module provides a subset of features provided by the system-wide sops-nix since features like the creation of the ramfs and changing the owner of the secrets are not available for non-root users.
 
 Instead of running as an activation script, sops-nix runs as a systemd user service called `sops-nix.service`.
-And instead of decrypting to `/run/secrets`, the secrets are stored decrypted to `$XDG_RUNTIME_DIR/secrets` that is located on a tmpfs or similar non-persistent filesystem. Additionally secrets are symlinked to the user home-directory in the `.secrets`-directory which is used as reference 
-for the `.path` value in nix. Because of that, the home-manager option `home.homeDirectory` is used to determinate the home-directory on evaluation, 
+And instead of decrypting to `/run/secrets`, the secrets are stored decrypted to `$XDG_RUNTIME_DIR/secrets` that is located on a tmpfs or similar non-persistent filesystem.
```

Comment:
On my system it is `$XDG_RUNTIME_DIR/secrets.d`
```
ll $XDG_RUNTIME_DIR/secrets.d
drwxr-x--x   - sedlund 23 Apr 16:14  21/
.rw------- 123 sedlund 23 Apr 16:14  age-keys.txt
```

I was looking for clarification with my comments on the prior PR and was going to submit a PR like this to this to reword this whole section for clarity.

For this section I would rewrite it:

> While the sops-nix _system_ module decrypts secrets to the system non-persistent `/run/secrets`, the _home-manager_ module places them in the users non-persistent `$XDG_RUNTIME_DIR/secrets.d`.

Reply:
Ok. We can also go with your version.